hibike/hibikeDevices.json

```diff
@@ -67,6 +61,26 @@
                                                      ]
     },
 
+
+    {"id": 10,     "name": "YogiBear",      "params": [
```

Comment:
If @karthik-shanmugam thinks the YogiBear PR is ready for merging, I'd rather merge that first than include this diff here.